portal-api/src/v1/gef/controllers/application.controller.js

```diff
@@ -232,7 +232,11 @@ exports.changeStatus = async (req, res) => {
       existingApplication,
     );
 
+    // portalActivities array with added submission activity
+    const portalActivities = await submissionPortalActivity(existingApplication);
+
     applicationUpdate = {
+      portalActivities,
```

Comment:
Instead of adding it in here, could/should we add the new activities data as part of `addSubmissionData()`?

Reply:
done